pkg/catalog/catalog.go

```diff
@@ -67,6 +72,10 @@ func Exists(client *occlient.Client, componentType string) (bool, error) {
 // VersionExists checks if that version exists, returns true if the given version exists, false if not
 func VersionExists(client *occlient.Client, componentType string, componentVersion string) (bool, error) {
 
+	// Loading status
+	s := util.LogStatus("Checking image version")
```

Comment:
should say component version, user might not know that there is some image

Reply:
I would even say that user doesn't need to know about this.

Reply:
same here. Agree with @kadel the user doesn't need to know about this